### DISCUSSION TITLE
Yeni sürümde mobil alt ikonlara anasayfa ikonu

### DISCUSSION THREAD

**Discussion by EchoBilisim** [realsuayip/django-sozluk#36] [Ideas]
Merhaba güzel yazılım olmuş mobilden siteye girince alt tarafta ikonların olduğu bölüm anasayfa ikonu eklense daha kullanışlı olur her seferinde logoya tıklamak yerine orda tıkladımı anasayfa yönlenir.
![IMG_20220120_075744](https://user-images.githubusercontent.com/17055352/150276297-14853d97-c0d1-4d49-801e-25616b352101.jpg)

**Comment by realsuayip** [comment]
selamlar, bu konuyu düşündük fakat kullanıcılar anasayfada o kadar da zaman geçirmiyorlar, o yüzden öyle bir butonun orada işlevsiz kalacağına karar verdik.

**Reply by EchoBilisim** [reply]
Merhaba zaman geçirmek için değil herhangi bir bölümdeyken tıklayıp ana sayfaya gitmek için kullanılması için iyi olacaktır bence bu sadece sizde her yazılımda olması gereken birşey ve bir çok yazılımda bu olması gereken birşey ve kullanım olarak daha iyi hale getiriliyor genede karar sizin siz daha iyi bilirsiniz
